config/locales/controllers/en.yml

```
@@ -132,7 +132,9 @@ en:
     tos_faq:
       page_title: Terms of Service FAQ
   inbox:
+    must_select_item: Please select something first.
     page_title: "%{user} - Inbox"
+    update_success: Inbox successfully updated.
```

Not required, but it would be nice to scope these by action instead, like with the other controllers. And then the success message key can just be `success`.

---

For the success action, that's easy enough. 

For `must_select_item` I'm not sure what to use as a better key. The only similar key that I could find in the `locales/controllers/en.yml` file was for removing pseuds and it was my original inspiration (`must_select_pseud`).

---

I think Bilka means something like this:
```suggestion
inbox:
    page_title: "%{user} - Inbox"
    update:
        must_select_item: Please select something first.
        success: Inbox successfully updated.
```

---

Yeah, that's what I mean. (Ideally also for the page_title in `show`.)

For the message, `Please select at least one comment first.` would be good.

---

Thank you @lydia-theda and @Bilka2 for the clarification. I've updated as per your feedback, and I updated the text too!